### PR TITLE
Output code of the signal in tracking view legend

### PR DESCRIPTION
**Pull request by pmiettinen**

### CODE REVIEW

File: piksi_tools/console/tracking_view.py [hunk 1]
```diff
@@ -88,9 +91,9 @@ def tracking_state_callback(self, sbp_msg, **metadata):
     for n, k in enumerate(self.states):
       s = sbp_msg.states[n]
       prn = s.sid.sat
-      if (s.sid.code == 0):
+      if (s.sid.code == 0 or s.sid.code == 1):
```
Comment (jacobmcnamee) [inline piksi_tools/console/tracking_view.py]: This was correct as is. GPS L1 is encoded as prn-1 over SBP for backwards compatibility. Other code types are not converted.

Comment (pmiettinen) [inline piksi_tools/console/tracking_view.py]: But the PRNs are matching on the legend with this clause?

Comment (jacobmcnamee) [inline piksi_tools/console/tracking_view.py]: Oh wait, I lied, sorry:
```
  /* Maintain legacy compatibility with GPS PRN encoding. Sat values for other
   * constellations are "real" satellite identifiers.
   */
  if (code_valid(sid.code) &&
     (code_to_constellation(sid.code) == CONSTELLATION_GPS)) {
    sid.sat += GPS_FIRST_PRN;
  }
```
It's all GPS sats.

**Comment by denniszollo** [comment]
LGTM - please merge.  With Piksi tools pull requests please include a screenshot when relevant in general.

**Pull request merged**